packages/formation-react/src/components/ErrorableCheckbox/ErrorableCheckbox.jsx

```diff
@@ -51,6 +51,7 @@ class ErrorableCheckbox extends React.Component {
           </span>
         )}
         <input
+          aria-labelledby={this.props.ariaLabelledBy}
```

Comment:
As long as this `aria-labelledby` attribute string matches the ID being passed into the heading above it, and those IDs are unique, this should be fine. I'm not able to get this running locally to verify it, but looking at the code this seems logical and correct. I'll plan to test in Staging when I am back in the office.

Reply:
@jbalboni , @1Copenut , not hitting a solid brick wall trying to remove the label, but quite a thick hedge...

This is a _custom_ checkbox, and relies on the label for both displaying a “fake” checkbox (with CSS label::before) and more-importantly the click-handling as well.

The User doesn't really click the input element itself, which is hidden (with CSS opacity:0).  The click is actually on the label (empty or not) and relies on its for attribute to change the hidden-input value.

I'm just wondering though whether it's worth a major refactor (both styling and functionality) just to remove the label element for this use-case (which seems an edge one to me).  The styling is not just within formation, but rides on top of external uswds stylesheets, making any tweaks more brittle than I'd like.

What are your thoughts?  Any harm done leaving the empty label?

Reply:
Ah, I had forgotten that that's how the checkbox was implemented. No, I don't think it's worth trying to refactor that, as long as the checkbox seems work for fine when the `label` has no text content.

Reply:
Thx, @jbalboni !  Leaving empty label then for this.